[BUGFIX] Prevent SiteNotFoundException in page module

astehlik opened the pull request:
When accessing a page with the page module that is not inside
a configured site you currently get an exception:

No site found in root line of page xxx.

This Exception is now catched and the inclusion of the
Siteimporve JS in the page module is skipped when no
domain is available.

astehlik commented:
Sorry @mabolek 

GitHub Actions did not like the slash in the branch name so I had to recreate the pull request.

astehlik commented:
I had to replace some hardcoded paths in the build config to make the Actions work.

At the end I had some strange random build errors. After some retries everything is green now:

https://github.com/Intera/typo3-extension-pxa_siteimprove/actions/runs/358077217

mabolek reviewed (approved):
Thank you for this PR! I'm not sure why the paths didn't work, but we'll look into that on our side.

astehlik commented:
@mabolek I guess the problem with the paths was, that we renamed the our forked repository. Therefore the paths did not work any more. I replaced them with a GitLab variable so this should always work now :)
